metta/metta_io.py

```diff
@@ -21,7 +21,8 @@ def archive_train_test(train_config,
                        df_test,
                        directory='.',
                        format='hd5',
-                       overwrite=False):
+                       overwrite=False,
+                       write_index=True):
```

Comment:
Yep. When reading in a csv from a path, the index will just be row numbers, and we do not want those written.